AWS/aws-cloudtrail/_meta/smart-descriptions.json

```diff
@@ -25,12 +25,15 @@
     ]
   },
   {
-    "value": "{source.address} called {event.action} method from {event.provider} ",
+    "value": "{source.address} called {event.action} method from {event.provider} on cluster {aws.cloudtrail.cluster_name}",
     "conditions": [
       {
         "field": "action.type",
         "value": "AwsApiCall"
       },
+      {
+        "field": "aws.cloudtrail.cluster_name"
+      },
```

Comment:
Could you duplicate this smart-description? However, the change breaks smart-descriptions for events without cluster_name.

Reply:
Yes, ok, I will duplicate this smart-description, I didn't find event with         
"field": "action.type",
"value": "AwsApiCall"
without cluster_name but you find one I guess